### ISSUE TITLE
[BUG] Context menu active color effect

### ISSUE THREAD

**Issue by cylixlee** [RichardLuo0/vscode-frosted-glass-theme#5]
Hi Richard it's me again. I like to change themes often to resist visual fatigue. Frost Glass offers corresponding blurred UI under both light themes and dark themes and that's excellent.

Here's what it looks like when I use a dark theme:
![image](https://github.com/RichardLuo0/vscode-frosted-glass-theme/assets/153902646/d5050899-aa7d-4b35-8fa9-6d5ec12f8c6d)

And when I use `Ra Spring` light theme:
![image](https://github.com/RichardLuo0/vscode-frosted-glass-theme/assets/153902646/71fb77fc-ac59-4509-a150-2aef07dd6c67)

It has the correct effect. Strange things happen when I use the `Brackets Light Pro` light theme:
![image](https://github.com/RichardLuo0/vscode-frosted-glass-theme/assets/153902646/1111f2b0-37c5-4452-8572-787e35c45d71)

The active menu item has the **white** color on the **text**, making it hard to recognize characters in such a light theme. I guess, since I don't know the underlying mechanism of Frost Glass, that's probably because something went wrong when **detecting** it's currently light or dark theme.

Exaust all possible color themes is apparently impossible, so I'll appreciate it if you add a option in the settings to change the active color or something. I really like `Brackets Light Pro` and a workaround is enough for me.

**Comment by RichardLuo0** [comment]
I am working on it

**Comment by RichardLuo0** [comment]
You can try adding the following code to your settings.json
```json
"workbench.colorCustomizations": {
    "[Your theme name]": {
        "menu.selectionForeground": "#000000"
    },
}
```
In the current version the effect color is the same as selectionForeground.

In the next release I will change it to be the same as `menu.selectionBackground`.
So you can customize the color with `workbench.colorCustomizations`.

Many themes use colors other than black or white for `menu.selectionBackground`. However in my experience, only black or white is good for the light effect. Thus you will need to try different combinations by yourself.

**Comment by cylixlee** [comment]
It works! I can now use my favorite light theme and favorite blur effect ☺️

**Comment by RichardLuo0** [comment]
I have released a new version (0.7.2).
After the update, `menu.selectionForeground` will control the text color when selected, `menu.selectionBackground` will control the light effects color. And there is still `frosted-glass-theme.revealEffect.opacity` that will control the transparency of light effect.  You will need to find suitable combinations for each theme.
